sentences/nb/_common.yaml

```
@@ -350,6 +350,18 @@ lists:
     range:
       from: 1
       to: 100
+  timer_half:
+    values:
+      - in: "halv"
+        out: 30
+      - in: "halvt"
+        out: 30
+  timer_quarter:
+    values:
+      - in: "[et |ett ]kvarter"
+        out: 15
+      - in: "[3|tre] kvarter"
```

Should not be optional, need to use `(` and not `[`

https://developers.home-assistant.io/docs/voice/intent-recognition/template-sentence-syntax/#sentence-templates-syntax

```suggestion
      - in: "(3|tre) kvarter"
```